package/aws/README.md

```diff
@@ -91,13 +112,13 @@ Next in the file, we should declare a version for our resource and lay out the s
 ```
 
 We should then add properties of the resources under the third `properties:` we defined above. We can copy and paste 
-these from the source code of the resource in aws-provider (using the desired release tag). For example, `https://raw.githubusercontent.com/crossplane/provider-aws/v0.20.1/package/crds/s3.aws.crossplane.io_buckets.yaml` and copy the forProvider.properties
+these from the source code of the resource in aws-provider (using the desired release tag). For example, https://raw.githubusercontent.com/crossplane/provider-aws/v0.20.1/package/crds/s3.aws.crossplane.io_buckets.yaml and copy the forProvider.properties. The full list of resource source code can be found here: https://github.com/crossplane/provider-aws/tree/v0.20.1/package/crds

```

Comment:
The full list of resource source code can be found here: https://github.com/crossplane/provider-aws/tree/[release-tag-name]/package/crds. Note: we use v0.20.1 in the example for [release-tag-name]

Reply:
Good suggestion. I've omitted the tag from the URL completely and used master, but made a note to set the appropriate tag based on crossplane.yaml. That way we don't have to update the readme each time we bump the minimum version